packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -126,8 +126,12 @@ If your API is consumed from another machine or service (instead of users), you
 
 If you are using [Postman](/code/rest/) and the Client Credentials flow, you can use an API call of the form `https://${yourOktaDomain}/oauth2/default/v1/token`, with an Authorization header set, and you will get a response containing an access token.
 
+<div class="full border">
+
 ![Postman showing a token request post response including an access token](/img/postman-post-response.png)
```

Comment:
I would set this to 3/4 size, rather than full. I suggest improving the image: removing the actual dev org; user-name value; and access token value. Blurring the latter two would be a good solution..

Reply:
done; updated to three-quarter, and sensitive data blurred using the snagit guidelines on confluence.